crates/hyperqueue/src/dashboard/ui/screen/controller.rs

```diff
@@ -0,0 +1,37 @@
+use crate::dashboard::events::{DashboardEvent, ScreenChangeEvt};
+use tako::WorkerId;
+use tokio::sync::mpsc::UnboundedSender;
+
+#[derive(Clone, Copy)]
+pub enum DashboardScreenState {
```

Comment:
Let's rename this to `ChangeScreenCommand`, this should represent an action that changes the screen.